### DISCUSSION TITLE
Connecting IoT module (SIM7000E) to srsEPC and srsRAN testbed

### DISCUSSION THREAD

**Discussion by ThapeloMakhalanyane** [srsran/srsRAN_Project#754] [General Help]
Hi all,

I've just built a testbed with srsEPC and srsRAN. I'm trying to connect SIM7000E IoT module to the testbed but I cannot establish a connection between the two. I believe I should change the E-UTRA band to one of the bands allocated to IoT (e.g band 28 (700MHz)), however, I have no idea how to go about it. the enb.conf script doesn't give this option. Please advice on how to go about solving this issue.

Your assistance or advice would be greatly appreciated.

**Comment by pgawlowicz** [comment]
srsEPC does not work with the srsRAN 5G project.
